### PR TITLE
add xocodergit into members.yaml

### PR DESCRIPTION
**Pull request by xocodergit**
## I want to join the Istio org!

- [x] I have reviewed the [community membership guidelines](https://github.com/istio/community/blob/master/ROLES.md#member)
- [x] I have reviewed the [contribution guidelines](https://github.com/istio/community/blob/master/CONTRIBUTING.md)
- [x] I have enabled [2FA on my GitHub account](https://github.com/settings/security)
- [x] I have joined [Istio's Slack workspace](https://slack.istio.io)
- [x] I have written my company name or "Individual", if not affliated with a company, here: Alibaba Cloud
- [x] I have provided a link to at least one PR that I have successfully pushed to one of the Istio repos, here: https://github.com/istio/ztunnel/pull/1248#event-13840313430

### CODE REVIEW

File: org/members.yaml [hunk 1]
```diff
@@ -534,3 +534,4 @@ members:
 - zqzzq
 - zufardhiyaulhaq
 - zyfjeff
+- xocodergit
```
Comment (zirain) [inline org/members.yaml:537]: Alphabetical sort

**Comment by craigbox** [comment]
Please update the comment with your affiliation.

File: org/members.yaml (hunk 1 above)
Comment (xocodergit) [reply org/members.yaml]: done

**Comment by kfaseela** [comment]
@xocodergit : please add your company affiliation in the PR description as Craig suggested

**Review by kfaseela — APPROVED** [review]

**Comment by xocodergit** [comment]
> I have written my company name or "Individual", if not affliated with a company, here: Alibaba Cloud
company affiliation added

**Pull request merged**